.eslintrc.js

```diff
@@ -160,8 +161,7 @@ module.exports = {
     'react/state-in-constructor': 'off',
     // stylistic opinion. For conditional assignment we want it outside, otherwise as static
     'react/static-property-placement': 'off',
-    // Currently not in recommended ruleset but catches real bugs.
-    'react/no-unstable-nested-components': 'error',
```

Comment:
Why is this one removed? Is it a part of a default ruleset now?

Reply:
yes, there may be other redundant ones, but this rule was added while I was working on this PR and I remembered seeing the errors